app/assets/javascripts/discourse/tests/acceptance/modal/login-test.js

```diff
@@ -0,0 +1,14 @@
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { click, tab, visit } from "@ember/test-helpers";
+
+acceptance("Modal - Login", function () {
+  test("You can tab to the login button", async function (assert) {
+    await visit("/");
+    await click("header .login-button");
+    // you have to press the tab key twice to get to the login button
+    await tab({ unRestrainTabIndex: true });
+    await tab({ unRestrainTabIndex: true });
```

Comment:
TIL about the `unRestrainTabIndex` parameter. I guess we want to make sure an error is thrown if for some reason there's a tabindex greater than 0?

Reply:
Well we already have elements with a tab index greater than 1... that is why I implemented `unRestrainTabIndex `

https://github.com/discourse/discourse/blob/af30f9945ab4da3f73ff70ec44ecda6c685f364e/app/assets/javascripts/discourse/app/templates/modal/login.hbs#L57 https://github.com/discourse/discourse/blob/af30f9945ab4da3f73ff70ec44ecda6c685f364e/app/assets/javascripts/discourse/app/templates/modal/login.hbs#L39